### PR TITLE
Screens

### PR DESCRIPTION
**Pull request by kshitijthatte**
This PR only has the HTML and CSS screens for the video library app. It also has some basic routing.
* Homepage
* Login & Signup pages
* Playlists page
* The following two use the `Single Playlist Page`
  * Liked Videos 
  * Watch Later

### CODE REVIEW

File: src/pages/SinglePlaylist/HorizontalVideoCard.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const HorizontalVideoCard = () => {
+  return (
+    <div class="card card-badge horizontal-video-card">
+      <div class="card-horizontal">
+        <img class="card-img" src="https://picsum.photos/300/200" />
+        <button class="btn card-icon c-badge-right material-icons">
+          cancel
+        </button>
+        <div class="card-content">
+          <div class="card-title">Our Changing Planet</div>
```
Comment (Sen-442b) [inline src/pages/SinglePlaylist/HorizontalVideoCard.js]: You can replace the `class` attribute with  react `className`

File: src/pages/Signup/index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const Signup = () => {
+  return (
+    <div className="authentication-container">
+      <div className="card input-group">
+        <Link className="title-w-logo text-lg text-primary text-center" to="/">
+          <img src="/assets/logo-colored.svg" className="nav-logo" />
+          pawws - Sign Up
+        </Link>
+        <div className="name-input">
+          <input type="text" className="input" placeholder="First Name" />
```
Comment (Sen-442b) [inline src/pages/Signup/index.js:13]: Suggestion:- you can give `role` attribute to you input tag for better accessibility

File: src/pages/SinglePlaylist/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            : playlistTitle === "Watch Later"
+            ? "watch_later"
+            : "playlist_play"}
+        </span>
+        {playlistTitle}
+      </header>
+      <p className="text-sm">8 videos</p>
+      <div className="playist-video-container">
+        {videos.map(video => (
+          <HorizontalVideoCard />
```
Comment (Sen-442b) [inline src/pages/SinglePlaylist/index.js:22]: you can also add key  unique key to your component

**Review by Sen-442b — APPROVED** [review]
- You have a clean folder struct 
-  classNames and vars are descriptive as well
-  UI looks minimal and clean to me 

also you can  keep the `App.js`  in  `components`  directory as it is also a component

File: src/App.js [hunk 4]
```diff
@@ -1,10 +1,32 @@
 import "./styles.css";
+import { Route, Routes } from "react-router-dom";
+import NavDrawer from "./components/NavDrawer";
+import Home from "./pages/Home/";
+import Login from "./pages/Login";
+import Signup from "./pages/Signup";
+import Playlists from "./pages/Playlists";
+import SinglePlaylist from "./pages/SinglePlaylist";
```
Comment (MiheerTamkhane) [inline src/App.js]: Barrel export will help here

**Review by MiheerTamkhane — APPROVED** [review]
To me code is clean and readable, the UI is also nice
Good to merge

File: src/pages/SinglePlaylist/HorizontalVideoCard.js (hunk 1 above)
Comment (kshitijthatte) [reply src/pages/SinglePlaylist/HorizontalVideoCard.js]: Thanks for pointing it out, will update that.

File: src/pages/Signup/index.js (hunk 2 above)
Comment (kshitijthatte) [reply src/pages/Signup/index.js:13]: Sure will have a look at it.

File: src/pages/SinglePlaylist/index.js (hunk 3 above)
Comment (kshitijthatte) [reply src/pages/SinglePlaylist/index.js:22]: Will do it, when I'll get the data from the DB.

File: src/App.js (hunk 4 above)
Comment (kshitijthatte) [reply src/App.js]: Thanks for the suggestion, instead I'll separate the routes to make the App.js cleaner.

**Pull request merged**